Expensieve/Expensieve
# Lucascontreras/fe service dashboard

## Overview

**Tailwind config and components**


**Description**
Tailwind is configured and working in the frontend, also the service dashboard is drafted with placeholder values for now.

**Steps to Validate Feature**

1. Install all the packages, build and run `npm run start-dev`
2. See the home view in the browser with the service dashboard
